### DISCUSSION TITLE
How do I enable my webiste at user.github.io??

### DISCUSSION THREAD

**Discussion by johnwry** [alshedivat/al-folio#307] [Q&A] [answered]
I have set up a blog by creating a repository /blog and i have the website working at username.github.io/blog/ 

What I would like to know how to do is have it so that username.github.io gives me a welcome page. Right now it's giving me a 404. 

Thanks in advance.

**Comment by alshedivat** [comment, marked as answer]
(sorry, looks like this question was posted in the wrong discussion category, so no one noticed it for a while)

you would need to rename your website repository from `blog` to `username.github.io` and follow the steps in https://github.com/alshedivat/al-folio/#deployment for personal webpages. lmk if you run into any issues.
